src/languages/vbnet.js

```diff
@@ -1,73 +1,160 @@
-/*
-Language: Visual Basic .NET
-Description: Visual Basic .NET (VB.NET) is a multi-paradigm, object-oriented programming language, implemented on the .NET Framework.
-Author: Poren Chiang <ren.chiang@gmail.com>
-Website: https://docs.microsoft.com/en-us/dotnet/visual-basic/getting-started/
-*/
+/**
+ * Language: Visual Basic .NET
+ * Description: Visual Basic .NET (VB.NET) is a multi-paradigm, object-oriented programming language, implemented on the .NET Framework.
+ * Authors: Poren Chiang <ren.chiang@gmail.com>, Jan Pilzer
+ * Website: https://docs.microsoft.com/dotnet/visual-basic/getting-started/
+ */
 
 export default function(hljs) {
+  /**
+   * Character Literal
+   * Either a single character ("a"C) or an escaped double quote (""""C).
+   */
+  const CHARACTER = {
+    className: 'string',
+    begin: /"(""|[^/n])"C\b/
+  };
+
+  const STRING = {
+    className: 'string',
+    begin: /"/,
+    end: /"/,
+    illegal: /\n/,
+    contains: [
+      {
+        // double quote escape
+        begin: /""/
+      }
+    ]
+  };
+
+  /**
+   * Date Literals consist of a date, a time, or both separated by whitespace, surrounded by #.
+   */
+  const DATE = {
+    className: 'literal',
+    variants: [
+      {
+        // Date: #M/D/YYYY# or #YYYY-MM-DD#
+        begin: /# *((\d+[-/]){2}\d+) *#/
```

Comment:
Too broad be specific and match as precisely as we can:

```
// MDY
\d{1,2}-\d{1,2}-\d{4}
```

Etc.

Reply:
Split and mores specific regexes.